tests/e2e/chrome_test.go

```diff
@@ -0,0 +1,77 @@
+package e2e
+
+import (
+	"github.com/stretchr/testify/suite"
+	"github.com/tebeka/selenium"
+	"log"
+	"testing"
+)
+
+type ChromeTestSuite struct {
```

Comment:
Can we have a single TestSuite to which we pass the browser name and it appropriately sets up the webdriver and runs the tests?

Reply:
I got an error saying that browserTestSuite already has a test definitions

Reply:
Refer the changes [here](https://github.com/padaiyal/sanitizer/pull/48/files).